content/src/fieldConfig/profile-info-oscar.json

```diff
@@ -50,7 +50,7 @@
       },
       "nexusDbaName": {
         "placeholder": "",
-        "header": "",
+        "header": "### Doing Business As (DBA) New Jersey Business Name",
```

Comment:
why are these not empty string? Shouldn't Poppy & Oscar never have the DBA profile field? They should never be used, right?

Reply:
I thought they had to look the same in all three profile info files, i will update that to be empty.